### PR TITLE
Video page

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

I have added ID based dynamic video pages to my video library app.

The videos under "Must Watch" are of the same category as the video thats been selected.
For now I have setup only 2 categories (Computer Programming and Entertainment) and filtering is done on the video pages on the basis of that.

Please review my code and let me know if any changes need to be made.

Link : https://water-videos-git-video-page-shlok30.vercel.app/

Thanks in advance,
Shlok

### CODE REVIEW

File: src/pages/video.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    
+    const {videoId} = useParams()
+    
+    const {videoLibrary} = useVideos()
+
+    const {title,creator,description,views,categoryName} = videoLibrary.find((video) => video["_id"] === videoId)
+
+    const videosOfSameCategory = videoLibrary.filter((video) => video.categoryName === categoryName && video["_id"] !== videoId)
+
+    //console.log(selectedVideoDetails)
```
Comment (chanchal-panpaliya) [inline src/pages/video.js:17]: remove console

**Review by chanchal-panpaliya — APPROVED** [review]
1. code looks clean 
2. proper naming 
3. good UI
4. good folder structure

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -24,4 +24,10 @@ export const categories = [
     description:
       "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
   },
+  {
+    _id: uuid(),
+    categoryName: "Entertainment",
+    description:
+      "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
```
Comment (nikhil-sharma-nks) [inline src/backend/db/categories.js:31]: It's not an error but you should consider writing relevant description here.

File: src/pages/video.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+const VideoPage = () => {
+    
+    const {videoId} = useParams()
+    
+    const {videoLibrary} = useVideos()
+
+    const {title,creator,description,views,categoryName} = videoLibrary.find((video) => video["_id"] === videoId)
+
+    const videosOfSameCategory = videoLibrary.filter((video) => video.categoryName === categoryName && video["_id"] !== videoId)
```
Comment (nikhil-sharma-nks) [inline src/pages/video.js:15]: if videosOfSameCategory is being used at other places too then you should consider making it a util function

**Review by nikhil-sharma-nks — APPROVED** [review]
You have implemented good coding practice that's really good. Your video library looks good! Good work👍

**Pull request merged**